boards/shields/xenvm_dom0/Kconfig.defconfig

```diff
@@ -6,4 +6,9 @@ if SHIELD_XENVM_DOM0
 config HEAP_MEM_POOL_SIZE
 	default 2097152
 
+# it is needed at least 16M for grant tables
```

Comment:
I'm not sure how Xen determines the size of Grant table for Domain-0, probably it can be smaller. As I know for DomUs number of gnt frames is configured via dom.cfg. Maybe @lorc will clarify this moment

Reply:
at least this config only for dom0 and probably comment isn't clear, this is needed for grant table driver
probably, I can try to set default value using value of size from dts